tests/rust/log_benchmark/src/image.rs

```diff
@@ -0,0 +1,71 @@
+use rerun::{
+    datatypes::TensorBuffer,
+    datatypes::{TensorData, TensorDimension},
+    external::re_types_core::ArrowBuffer,
+};
+
+// About 1gb of image data.
+const IMAGE_DIMENSION: u64 = 16_384;
+const IMAGE_CHANNELS: u64 = 4;
+
+// How many times we log the image.
+// Each time with a single pixel changed.
+const NUM_LOG_CALLS: usize = 4;
```

Comment:
Nit: the PR description says 5 images, but this looks like 4?

Reply:
ah. I simply misremembered!